src/lib/processImage.js

```diff
@@ -9,11 +9,21 @@ module.exports = (content, query) => {
         throw new Error(`Method "${methodName}" is undefined`)
       }
 
-      const params = (
+      const paramsBefore = (
         Array.isArray(query[methodName]) ?
           query[methodName] :
           [query[methodName]]
       )
+
+      const params = paramsBefore.map((key) => {
+        var parsed = parseInt(key)
```

Comment:
can run `npm run lint`? It will not pass
and also I think we should use `Number()` instead of `parseInt()`, because
1. if user input any string other than number then it is their mistake
2. it will remove all decimal places, it is needed for params like `gamma`